modules/rosa-view-cloudwatch-logs.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-view-cloudwatch-logs_{context}"]
+= Viewing forwarded logs
+
+.Prerequisites
+
+* The `cluster-logging-operator` add-on service is installed.
+* The Amazon CloudWatch add-on service is installed.
+
+.Procedure
+
+To view logs that have been forwarded to Amazon Web Services (AWS) from {product-title}:
```

Comment:
Same about not having a lead-in sentence to the bullets after the .Procedure heading. Any sort of intro like this should be a full sentence and above the Prerequisites.

Reply:
Done.